contracts/utils/cryptography/ECDSA.sol

```diff
@@ -33,8 +33,11 @@ library ECDSA {
     error ECDSAInvalidSignatureS(bytes32 s);
 
     /**
-     * @dev Returns the address that signed a hashed message (`hash`) with
-     * `signature` or error string. This address can then be used for verification purposes.
+     * @dev Returns the address that signed a hashed message (`hash`) with `signature` or an error. This will not
+     * return address(0) without also returning an error description. Errors are documented using an enum (error type)
+     * and a bytes32 providing additional information about the error.
+     *
+     * If no error is returned, then the address can then be used for verification purposes.
```

Comment:
“then the address can then be” -> “then the address can be”?

Reply:
```suggestion
     * If no error is returned, then the address can be used for verification purposes.
```